app/scripts/controllers/metametrics.js

```diff
@@ -443,13 +443,18 @@ export default class MetaMetricsController {
     }
   }
 
+  // Track metametrics event by creating a fragment for it.
+  trackEvent(payload, options) {
+    this.createEventFragment({ ...payload, ...options });
```

Comment:
this is not sufficient, as we will need to immediately *finalize* the fragment. Otherwise the fragment is just created not tracked.

Reply:
I implemented the change @brad-decker